src/plugins/intel_cpu/tests/functional/subgraph_tests/src/fq_layer_dq_bias.cpp

```diff
@@ -0,0 +1,118 @@
+// Copyright (C) 2023 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "lpt_ngraph_functions/markup_bias_function.hpp"
+#include "ngraph_functions/builders.hpp"
+#include "ngraph_functions/utils/ngraph_helpers.hpp"
+#include "shared_test_classes/base/layer_test_utils.hpp"
+#include "shared_test_classes/base/ov_subgraph.hpp"
+#include "test_utils/cpu_test_utils.hpp"
+#include "test_utils/fusing_test_utils.hpp"
+
+using namespace ngraph;
+using namespace ov::test;
+using namespace CPUTestUtils;
+using namespace InferenceEngine;
+
+namespace SubgraphTestsDefinitions {
+using FQLayerDQBiasParams = std::tuple<InputShape, std::string>;
+
+class FQLayerDQBias : virtual public SubgraphBaseTest,
+                      public CpuTestWithFusing,
+                      public testing::WithParamInterface<FQLayerDQBiasParams> {
+public:
+    static std::string getTestCaseName(testing::TestParamInfo<FQLayerDQBiasParams> obj) {
+        InputShape input_shape;
+        std::string layer_type;
+        std::tie(input_shape, layer_type) = obj.param;
+
+        std::ostringstream result;
+        result << "IS=(" << CommonTestUtils::partialShape2str({input_shape.first}) << ")_TS=(";
+        for (const auto& item : input_shape.second) {
+            result << CommonTestUtils::vec2str(item) << "_";
+        }
+        result << ")_layer_type=" << layer_type;
+        return result.str();
+    }
+
+protected:
+    void SetUp() override {
+        InputShape input_shape;
+        std::string layer_type;
+        std::tie(input_shape, layer_type) = GetParam();
+
+        targetDevice = CommonTestUtils::DEVICE_CPU;
+        fusedOps = std::vector<std::string>{"Add"};
+        std::tie(inFmts, outFmts, priority, selectedType) = CPUSpecificParams{{}, {}, {}, CPUTestsBase::any_type};
+        std::unordered_map<std::string, std::string> ngraph_type_to_plugin_type{
+            {"Convolution", "Convolution"},
+            {"GroupConvolution", "Convolution"},
+            {"ConvolutionBackpropData", "Deconvolution"},
+            {"MatMul", "MatMul"},
+            {"MatMulWithConstant", "FullyConnected"},
+        };
```

Comment:
Is not it better to use `static const std::unordered_map<std::string, std::string> ngraph_type_to_plugin_type`?

Reply:
Applied in #16856
Also a short description added for the test class.